Lib/test/test_locale.py

```diff
@@ -382,7 +382,7 @@ def setUp(self):
         "musl libc issue on Emscripten/WASI, bpo-46390"
     )
     def test_strcoll_with_diacritic(self):
-        self.assertLess(locale.strcoll('à', 'b'), 0)
+        self.assertGreater(locale.strcoll('à', 'b'), 0) # tested on CPython
```

Comment:
we normally don't edit CPython originated code without sufficient reason.
If it fails, we usually prefer to keep it as `@unittest.expectedFailure`